pkg/console/subresource/route/route_test.go

```diff
@@ -22,290 +20,6 @@ const (
 	tlsCertificate = "certificate"
 )
 
-func TestDefaultRoute(t *testing.T) {
```

Comment:
Could you explain the decision to remove these tests rather than update them?

Reply:
since we moved from assembling the route manifest via function, to using the bindata I thought that it doesnt make sense to have those tests anymore.